src/sentry/monitors/clock_dispatch.py

```diff
@@ -85,6 +98,91 @@ def _make_reference_ts(ts: datetime):
     return int(ts.replace(second=0, microsecond=0).timestamp())
 
 
+def _evaluate_tick_decision(tick: datetime):
```

Comment:
I think this should be behind a feature flag so that it's easy to disable if it causes a problem

Reply:
sounds good